src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {ThemeProvider} from '../ThemeProvider';
+import {Card} from '../Card';
+import {TextContainer} from '../TextContainer';
+import {Thumbnail} from '../Thumbnail';
+import {Stack} from '../Stack';
+import {Button} from '../Button';
+import {TextStyle} from '../TextStyle';
+
+export interface OrderDetailsProps {}
+
+export class OrderDetails extends React.PureComponent<OrderDetailsProps> {
+  render() {
+    return (
+      <ThemeProvider
+        theme={{
+          colorScheme: 'dark',
+          colors: {
+            surface: '#020c1d',
+            secondary: '#103560',
+            interactive: '#009973',
+          },
+        }}
+      >
+        <Card title="Order details">
+          <Card.Section title="Unfulfilled">
+            <Stack alignment="center" distribution="equalSpacing">
+              <Stack alignment="center">
```

Comment:
Am I overusing Stacks here? Wasn't sure how else to do vertical/horizontal alignment

Reply:
Not sure, this is how i would approach it to avoid extra styling changes. 🤔

Reply:
If you want to accomplish a similar layout with a single stack, you can use the `Stack.Item` subcomponent with the `fill` prop to have it take up as much space as possible.

```
<Stack alignment="center">
  <Thumbnail
    source="https://burst.shopifycdn.com/photos/black-leather-choker-necklace_373x@2x.jpg"
    alt="Black choker necklace"
  />
  <Stack.Item fill>
    <TextStyle variation="positive">Enamel pin</TextStyle>
  </Stack.Item>
  <span>$9.99</span>
</Stack>
```